components/ClubModal/Details.js

```diff
@@ -50,10 +50,15 @@ const Details = ({ size, applicationRequired, acceptingMembers }) => (
   </DetailsTag>
 )
 
+Details.defaultProps = {
+  applicationRequired: null,
+  acceptingMembers: false,
+}
+
 Details.propTypes = {
   size: PropTypes.string.isRequired,
-  applicationRequired: PropTypes.number.isRequired,
-  acceptingMember: PropTypes.bool.isRequired,
+  applicationRequired: PropTypes.number,
+  acceptingMembers: PropTypes.bool,
```

Comment:
`isAcceptingMembers` is a better name I think

Reply:
This is the way it is stored in the backend to my understanding—good flag which might be worth the refactor but I don't really want to do it in this PR if that's ok